### PR TITLE
clean up topic usage

### PR DESCRIPTION
**Pull request by emord**
Cleans up places where we were using document types instead of kafka topics to pass into pillows

@dannyroberts

### CODE REVIEW

File: corehq/pillows/case.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def get_case_to_elasticsearch_pillow(pillow_id='CaseToElasticsearchPillow', **kw
         index_info=CASE_INDEX_INFO,
         doc_prep_fn=transform_case_for_elasticsearch
     )
-    kafka_change_feed = KafkaChangeFeed(topics=[topics.CASE, topics.CASE_SQL], group_id='cases-to-es')
+    kafka_change_feed = KafkaChangeFeed(topics=topics.CASE_TOPICS, group_id='cases-to-es')
```
Comment (dannyroberts) [inline corehq/pillows/case.py]: I assume this accepts either [lists] or (tuples,) for `topics`?

**Review by dannyroberts — APPROVED** [review]
Approve

File: corehq/pillows/case.py (hunk 1 above)
Comment (emord) [inline corehq/pillows/case.py]: Yeah. we expand it before passing to the library https://github.com/dimagi/commcare-hq/blob/je/kafka-checkpoint-only/corehq/apps/change_feed/consumer/feed.py#L119

Comment (dannyroberts) [inline corehq/pillows/case.py]: 👍

**Pull request merged**